src/ConvertInputFormat/main.cpp

```diff
@@ -255,12 +255,18 @@ bool processErrors(std::unique_ptr<IdfParser> const &idf_parser, std::unique_ptr
         displayMessage(warning);
     }
     for (auto const &error : validation_errors) {
-        if (isDDY) {
-            if ((error.find("Missing required property 'Building'") != std::string::npos) ||
-                (error.find("Missing required property 'GlobalGeometryRules'") != std::string::npos)) {
+        bool const missing_building = error.find("Missing required property 'Building'") != std::string::npos;
+        bool const missing_geometry = error.find("Missing required property 'GlobalGeometryRules'") != std::string::npos;
+        // if we encountered one of the expected missing building/geometry errors, we should handle them special
+        if (missing_building || missing_geometry) {
+            if (isDDY) { // for DDY files just ignore it completely
                 continue;
             }
+            // for other IDFs, go ahead and emit a message, but don't trigger a failure
+            displayMessage(error);
+            continue;
         }
+        // and if it wasn't a missing building or missing geometry error, we will emit that as a proper error and fail
```

Comment:
Just handle missing building and geometry a little better now.  
 - totally ignore for DDY files
 - emit message, but succeed for IDFs

All other error types, emit and mark the error flag.